docs/metrics/confusion-matrix.md

```diff
@@ -35,7 +31,26 @@ imbalances.
 
 The implementation of a confusion matrix depends on whether the workflow concerns one or more classes.
 
-### Single-Class
+!!!warning
```

Comment:
This feels more like an `info` than warning?

Reply:
Updated to info. Looks more appropriate.